src/lib/utils.js

```diff
@@ -22,8 +25,31 @@ class Utils {
 
   }
 
-}
+  static async readJson(path) {
+    try {
+      return JSON.parse(
+        await fs.readFile(path, {encoding: 'utf8'})
+      );
+    } catch (error){
+        console.error(error);
+    }
+  }
+
+  static async getContext(){
+    return await Utils.readJson(path.resolve(path.resolve(), './src/lib/oa.jsonld'));
+  }
 
+  static ensureArray(input){
```

Comment:
Won't ensureArray(undefined) return [ensureArray] where you probably want it to return []?

Reply:
I will write a test for and then fix that in a new PR :)